curp/src/server/storage/rocksdb.rs

```diff
@@ -69,14 +79,23 @@ impl<C: 'static + Command> StorageApi for RocksDBStorage<C> {
 
         Ok((voted_for, entries))
     }
+
+    async fn new_snapshot(&self) -> Result<Box<dyn SnapshotApi>, StorageError> {
+        // TODO: delete outdated snapshot
+        // TODO: better snapshot file naming
+        let dir = self.data_dir.join(format!("snapshot-{}", Local::now()));
```

Comment:
`Local::now` looks like it returns a current system time. And the system time may be affected by NTP. I recommend using the monotonic time.

Reply:
Though I think it's almost impossible for two snapshots to be taken at the same system time, I've added a uuid to make sure it doesn't happen.